src/solvers/dgsem_p4est/dg_2d_parabolic.jl

```diff
@@ -303,6 +329,70 @@ function calc_gradient!(gradients, u_transformed, t,
     return nothing
 end
 
+# This version is called during `calc_gradients!` and must be specialized because the flux
+# in the gradient is {u} which doesn't depend on normals. Thus, you don't need to scale by 
+# 2 and flip the sign when storing the mortar fluxes back into surface_flux_values
```

Comment:
```suggestion
# This version is called during `calc_gradients!` and must be specialized because the 
# flux for the gradient is {u}, which doesn't depend on the outward normal. Thus, 
# you don't need to scale by 2 (e.g., the scaling factor in the normals (and in the 
# contravariant vectors) along large/small elements across a non-conforming 
# interface in 2D) and flip the sign when storing the mortar fluxes back 
# into `surface_flux_values`.
```